lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -1328,11 +1346,14 @@ function dosomething_user_info_form($form, &$form_state, $account) {
     '#default_value' => $zipcode,
   );
   $birthday = dosomething_user_get_field_birthdate(dosomething_user_get_field('field_birthdate', $account), 'm/d/Y');
+  $age = dosomething_user_get_age($birthday);
+
   $form['info']['birthdate'] = array(
     '#title' => "Birthdate",
     '#type' => 'textfield',
     '#default_value' => $birthday,
     '#description' => t("Format: M/D/YYYY"),
+    '#field_suffix' => 'User age: ' . $age, 
```

Comment:
@angaither do you think I should add `. ' years old.'` here so when it is posted on the user profile, it reads "User age: 20 years old." instead of "User age: 20" or is it explicit enough as is?